usecase/usecase-webapi-xwalk-tests/testscripts/steps/steps.py

```diff
@@ -12,6 +12,19 @@ def i_visit_frame(context, key):
     context.app.driver.switch_to_frame(key)
     assert True
 
+@step(u'I jump frame')
+def i_jump_frame(context):
+    context.app.driver.switch_to_default_content()
+    assert True
+
+@step(u'I should see nothing in "{key}"')
```

Comment:
This step had been redefined to "I should see nothing in "{attr}" attr of "{key}" area", please check if somewhere is still use it, if no, please remove it.